### PR TITLE
[gapi.people] Fix: `SearchContactsResult` should be optional

### PR DESCRIPTION
**Pull request by ianwang**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developers.google.com/people/api/rest/v1/people/searchContacts
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/gapi.people/gapi.people-tests.ts
Comment (ianwang) [inline types/gapi.people/gapi.people-tests.ts:21]: not related to the goal of this PR but removing quotes to make it more consistent with other gapi packages like  [this one](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/gapi.auth2/gapi.auth2-tests.ts) (gapi uses both tho 🤷 ). also quote on object props in this case doesn't seem necessary

File: types/gapi.people/gapi.people-tests.ts
Comment (joshuaobrien) [inline types/gapi.people/gapi.people-tests.ts:125]:
```suggestion
          appendPre('No contacts found.');
```

**Review by joshuaobrien — APPROVED** [review]
LGTM after suggestion is merged

**Comment by ianwang** [comment]
Ready to merge

**Review by joshuaobrien — APPROVED** [review]

**Comment by ianwang** [comment]
Ready to merge

**Pull request merged**
